pkg/query-service/tests/test-deploy/docker-compose.yaml

```diff
@@ -0,0 +1,97 @@
+version: "2.4"
+
+services:
+  clickhouse:
+    image: yandex/clickhouse-server:21.12.3.32
+    volumes:
+      - ./clickhouse-config.xml:/etc/clickhouse-server/config.xml
+    healthcheck:
+      # "clickhouse", "client", "-u ${CLICKHOUSE_USER}", "--password ${CLICKHOUSE_PASSWORD}", "-q 'SELECT 1'"
+      test: ["CMD", "wget", "--spider", "-q", "localhost:8123/ping"]
+      interval: 30s
+      timeout: 5s
+      retries: 3
+
+  alertmanager:
+    image: signoz/alertmanager:0.5.0
+    volumes:
+      - ./alertmanager.yml:/prometheus/alertmanager.yml
+    command:
+      - '--config.file=/prometheus/alertmanager.yml'
+      - '--storage.path=/data'
+
+# Notes for Maintainers/Contributors who will change Line Numbers of Frontend & Query-Section. Please Update Line Numbers in `./scripts/commentLinesForSetup.sh` & `./CONTRIBUTING.md`
+
+
+  query-service:
+    image: golang:1.17.8
```

Comment:
> golang:1.17.8

Replace it with `query-service:latest` as `make build-query-service-amd64` builds with `latest` tag by default unless overridden with `DOCKER_TAG`  env.

Reply:
Done